README.md

```diff
@@ -1 +1,79 @@
-# tic-tac-toe
\ No newline at end of file
+#  Tic Tac Toe OOP build with Ruby (Object Oriented Programming)
+* Original project specification at [The Odin Project] (https://www.theodinproject/lesseons/oop)
+* The purpose of the project is to master object oriented programming in ruby.
+
+# This project has four milestones
+- Project setup
+- User interface
+- Game logic
+- Instruction for players
+
+
+# Game Rule:
+Tic Tac Toe is Cool Math Games:
+Xs and Os is a paper-and-pencil game for two players, X, O:
+* Players take turns filing the spaces in 3X3 grid.
+* The player who succeeds in placing three of their marks in the horizontal, vertical, or diagonal or wins the game.
+
+the following example game is won by the first player, X:
+![example](https://upload.wikimedia.org/wikipedia/commons/thumb/1/1b/Tic-tac-toe-game-1.svg/1280px-Tic-tac-toe-game-1.svg.png)
+
+### Project Structure
+
+```bash 
+├── README.md
+├── bin
+│   └── main.rb
+├── lib
+     ├── player.rb
+     ├── game_logic.rb
+
+```
+
+## How to Run the Game
+
+
+
+```
+$ git clone https://github.com/Yors-git/tic-tac-toe
+```
+
+- Open your git bash and clone the repo.
+
+- go to the directory of the file and run ruby "main.rb" or "main"
+
+- Good job you are now playing Tic-Tac-Toe game. 👌🙌
```

Comment:
Before the instructions on how to play the game, please add a `Prerequisites` and `Installation` sections. In these sections you should instruct the reader on how to prepare their system to play the game: what software packages they need (such as `ruby`), where to find them and how to install them.